velox/exec/tests/RowContainerTest.cpp

```diff
@@ -47,29 +74,86 @@ class RowContainerTest : public exec::test::RowContainerTestBase {
 
     auto result = BaseVector::create(expected->type(), size, pool_.get());
     container.extractColumn(oddRows.data(), size, column, result);
-    EXPECT_EQ(size, result->size());
-    for (vector_size_t row = 0; row < size; ++row) {
-      if (row % 2 == 0) {
-        EXPECT_TRUE(result->isNullAt(row)) << "at " << row;
+    return result;
+  }
+
+  VectorPtr testExtractColumnRowNumbersForOddRows(
```

Comment:
```
s/testExtractColumnRowNumbersForOddRows/testSelectiveExtractColumnForOddRows/
```

Reply:
Leaving this name as is since it directly gives the name of the API being tested.